charts/rd-judicial-api/values.yaml

```
@@ -34,6 +34,7 @@ java:
     Clean_Elinks_Responses_Days: 30
     Del_Joh_Profiles_Years: 7
     Del_Joh_Profiles: false
+    DB_READER_USERNAME: "DTS CFT DB Access Reader"
```

this isn't going to be enough for single server, you need to have the database server name as well, I think this should work:

```suggestion
    DB_READER_USERNAME: "DTS CFT DB Access Reader@rd-judicial-api-postgres-db-{{ .Values.global.environment }}"
```

---

on flexible server you will want to drop the bit after the @ though

---

Thanks @timja did the changes as per your suggestions Thank you so much